packages/enclave-sdk/scripts/test.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+./scripts/build_fixtures.sh
+
+cargo test -- $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)